panther_analysis_tool/main.py

```diff
@@ -1694,8 +1694,19 @@ def setup_dynaconf() -> Dict[str, Any]:
 def dynaconf_argparse_merge(
         argparse_dict: Dict[str, Any], config_file_settings: Dict[str, Any]
 ) -> None:
+    # Set up another parser w/ no defaults
+    aux_parser = argparse.ArgumentParser(argument_default=argparse.SUPPRESS)
```

Comment:
This feels kinda janky, but this is the only way I could find to get the desired precedence without changing our arg parsing library 

based on [this](https://stackoverflow.com/a/45803037)

Reply:
i think the way we're getting input needs to be refactored at some point, so as long as this establishes the standard precedence of `file->env->arg` I think it's OK

Reply:
I'm taking this as implied approval